flooder/src/main.rs

```diff
@@ -505,7 +505,8 @@ fn create_connection_pool(
         .map(|urls| {
             urls.iter()
                 .map(|url| {
-                    create_custom_connection(url).expect("it should return initialized connection")
+                    create_custom_connection(url, false)
```

Comment:
Please consider naming this to variable or constant like `let not_secured = false`

Reply:
solved with `enum Protocol`